doc/releases/release-notes-4.1.rst

```diff
@@ -24,359 +47,177 @@ https://docs.zephyrproject.org/latest/security/vulnerabilities.html
 API Changes
 ***********
 
-Removed APIs in this release
-============================
+Removed APIs
+============
 
- * The deprecated Bluetooth HCI driver API has been removed. It has been replaced by a
+ * The Bluetooth HCI driver API has been removed. It has been replaced by a
```

Comment:
I feel like this should still say something like "The old HCI driver API", since we still have an HCI driver API (described in the next sentence).

Reply:
yeah I was trying to remove some wording duplication, reading this back I made this sound super weird, changed it to "The legacy Bluetooth HCI..."